src/list.json

```diff
@@ -251,6 +251,14 @@
         ],
         "description": "Contribute to ServiceNow to receive a free tshirt",
         "link": "https://developer.servicenow.com/blog.do?p=/post/hacktoberfest-2022/"
+      },
+      {
+        "organization": "StepZen",
```

Comment:
Please Re-Style it by using this command in the Repo Directory.
`git checkout <your branch>`
`curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2339735/patch | git am`
`git push`

Reply:
Not working

Reply:
by repo directory you mean the directory of the forked repo right?

Reply:
Ran the commands and output returnd: Applying: Restyled by jq, prettier-json but still checks failing

Reply:
Done @thefool76